src/renderer/components/+user-management-service-accounts/service-accounts-details.tsx

```diff
@@ -15,28 +15,33 @@ import { getDetailsUrl } from "../../navigation";
 import { KubeObjectDetailsProps } from "../kube-object";
 import { apiManager } from "../../api/api-manager";
 import { KubeObjectMeta } from "../kube-object/kube-object-meta";
+import { Icon } from "../icon";
 
 interface Props extends KubeObjectDetailsProps<ServiceAccount> {
 }
 
 @observer
 export class ServiceAccountsDetails extends React.Component<Props> {
   @observable secrets: Secret[];
+  @observable imagePullSecrets: Secret[];
 
   @disposeOnUnmount
   loadSecrets = autorun(async () => {
     this.secrets = null;
+    this.imagePullSecrets = null;
     const { object: serviceAccount } = this.props;
     if (!serviceAccount) {
       return;
     }
     const namespace = serviceAccount.getNs();
     const secrets = serviceAccount.getSecrets().map(({ name }) => {
-      const secret = secretsStore.getByName(name, namespace);
-      if (!secret) return secretsStore.load({ name, namespace });
-      return secret;
+      return secretsStore.load({ name, namespace });
```

Comment:
`secretStore.load` calls `this.getByName(..)` so doing that multiple times is redundant